rofl-utils/src/https.rs

```diff
@@ -0,0 +1,125 @@
+//! A very simple HTTPS client that can be used inside ROFL apps.
+//!
+//! This simple client is needed because Fortanix EDP does not yet have support for mio/Tokio
+//! networking and so the usual `hyper` and `reqwest` cannot be used without patches.
```

Comment:
```suggestion
//! networking and so the usual `hyper` and `request` cannot be used without patches.
```

Reply:
It really is https://crates.io/crates/reqwest :-)